statick_tool/config.py

```diff
@@ -95,6 +95,20 @@ def get_enabled_reporting_plugins(self, level: str) -> List[str]:
         """Get what reporting plugins are enabled for a certain level."""
         return self.get_enabled_plugins(level, "reporting")
 
+    @classmethod
+    def str_to_bool(cls, val: Optional[Union[str, Any]]) -> bool:
+        """Convert a string to a bool."""
+        if val is None:
+            return False
+        truth_values = ["y", "yes", "t", "true", "on", "1"]
+        false_values = ["n", "no", "f", "false", "off", "0"]
+        temp = val.lower()
+        if temp in truth_values:
+            return True
+        if temp in false_values:
+            return False
```

Comment:
Yea, it's not technically required, but i also like how it makes the intent clear. i debated throwing a ValueError if something not in one of the lists was entered and decided against it since that would change how statick currently functions and I don't mind evaluating anything not "truthy" as false.

Reply:
I like the current implementation. I don't think we need to throw anything, evaluating to False seems appropriate.